### DISCUSSION TITLE
MS Teams webhooks

### DISCUSSION THREAD

**Discussion by cipisek9** [thinkst/opencanary#363] [Feature Requests]
Hi, as you know, Microsoft will no longer support webhooks through Office 365 connectors starting August 15th (source: [Retirement of Office 365 Connectors within Microsoft Teams](https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/)).

Has anyone tried setting it up using workflows as mentioned in the article above? I attempted to use the provided schema, but unfortunately, the received message in the channel is empty.

`{
    "type": "object",
    "properties": {
        "dst_host": {
            "type": "string"
        },
        "dst_port": {
            "type": "integer"
        },
        "local_time": {
            "type": "string"
        },
        "local_time_adjusted": {
            "type": "string"
        },
        "logdata": {
            "type": "object",
            "properties": {
                "msg": {
                    "type": "object",
                    "properties": {
                        "logdata": {
                            "type": "string"
                        }
                    }
                }
            }
        },
        "logtype": {
            "type": "integer"
        },
        "node_id": {
            "type": "string"
        },
        "src_host": {
            "type": "string"
        },
        "src_port": {
            "type": "integer"
        },
        "utc_time": {
            "type": "string"
        }
    }
}`
<img width="727" alt="Screenshot 2024-07-16 at 14 42 27" src="https://github.com/user-attachments/assets/f4ed051a-fc00-4aaa-a77c-5d544a405588">

Thank you in advance.

**Comment by thinkst-az** [comment]
Thanks @cipisek9. We're working on updating all our MS Teams integrations and this one is one the list too. The old MS Team webhook won't work as you rightly point it requires a specific format.
